prow/scripts/cluster-integration/kyma-upgrade-gardener-kyma2-minor-versions.sh

```diff
@@ -83,15 +83,10 @@ function upgrade() {
     utils::save_psp_list "${ARTIFACTS}/kyma-psp.json"
 
     log::info "### Run post-upgrade tests"
-    gardener::post_upgrade_test_fast_integration_kyma -d "$kyma2_install_dir/${KYMA_SOURCE}/tests/fast-integration"
+    gardener::post_upgrade_test_fast_integration_kyma -d "${kyma2_install_dir}/tests/fast-integration"
 
     log::info "### waiting some time to finish cleanups"
     sleep 60
-
-    log::info "### Run pre-upgrade tests again to validate component removal"
-    export KYMA_SOURCE="${minor_release_versions[$((i))]}"
```

Comment:
To reviewer:

IMO, this step is not necessary, the old version fast-integration test already tested in  deploy_base step